dj_beneficiary/test_utils/models.py

```diff
@@ -0,0 +1,81 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from dj_beneficiary.models import (
+    BaseAgentAbstract,
+    BaseDistrictAbstract,
+    BaseFacilityAbstract,
+    BaseIndividualBeneficiaryAbstract,
+    BaseOrganizationBeneficiaryAbstract,
+    BaseProvinceAbstract,
+    BaseWardAbstract,
+    ImplementingPartner,
+    FacilityType
+)
+
+
+# Put your test models here
+class Province(BaseProvinceAbstract):
+    pass
+
+
+class District(BaseDistrictAbstract):
+    province = models.ForeignKey(
+        Province,
+        on_delete=models.CASCADE,
+    )
+
+
+class Ward(BaseWardAbstract):
+    district = models.ForeignKey(
+        District,
+        on_delete=models.CASCADE,
+    )
+
+
+class Facility(BaseFacilityAbstract):
+    district = models.ForeignKey(
+        District,
+        default=1,
+        on_delete=models.CASCADE,
+        help_text=_("District in which the facility is located"),
+        max_length=250,
+    )
+    facility_type = models.ForeignKey(
+        FacilityType,
```

Comment:
this model `FacilityType` DOES not need any abstructation @jmnda-dev 
these should be left as they are 
the only models we need to abstract are 
- OrganizationBeneficiary
- IndividualBeneficiary

so for District, Ward, Province
let's have a single abstract class called `AbstractLocationBoundary`
and just have name there
and then in PPCR we can define district, Ward and Province inheriting from `AbstractLocationBoundary`

Please take care of this review in more details as it helps clean up